index.html

```diff
@@ -238,176 +234,101 @@
     page: 0
   };
 
+var tbody = d3.select("tbody");
+
 function updateTable() {
-    // Make a copy of the data
-    var data = occupations.slice();
```

Comment:
So we still need to transform the data before we use it to create our table. We need to keep this copy, and move lines 248-268 above line 243, so that we can bind with the .data() call our transformed data, not the original.